app/src/main/java/org/schabi/newpipe/about/AboutActivity.java

```diff
@@ -71,10 +56,21 @@ protected void onCreate(Bundle savedInstanceState) {
         getSupportActionBar().setDisplayHomeAsUpEnabled(true);
         // Create the adapter that will return a fragment for each of the three
         // primary sections of the activity.
-        mSectionsPagerAdapter = new SectionsPagerAdapter(getSupportFragmentManager());
+        /*
```

Comment:
This is also not good, because previously this commend was part of the jdoc documentation, now it will not show up anymore.